lib/pos_blockchain.go

```diff
@@ -481,9 +482,164 @@ func (bc *Blockchain) pruneUncommittedBlocks(desoBlock *MsgDeSoBlock) error {
 // commitGrandparents commits the grandparent of the block if possible.
 // Specifically, this updates the CommittedBlockStatus of its grandparent
 // and flushes the view after connecting the grandparent block to the DB.
-func (bc *Blockchain) commitGrandparents(desoBlock *MsgDeSoBlock) error {
-	// TODO: Implement me.
-	return errors.New("IMPLEMENT ME")
+func (bc *Blockchain) commitGrandparents() error {
```

Comment:
consider a more self-descriptive function name that describes exactly what the function does
```suggestion
func (bc *Blockchain) runCommitRuleOnBestChain() error {
```